timesketch/lib/analyzers/sigma_tagger.py

```diff
@@ -125,9 +125,9 @@ def run(self):
                                 logger.info(
                                     '[sigma] Generated query {0:s}'
                                     .format(sigma_rule))
-                                sum_of_tagged_events = self.run_sigma_rule(
+                                tagged_events_counter = self.run_sigma_rule(
                                     sigma_rule, tag_name)
-                                tags_applied[tag_name] += sum_of_tagged_events
+                                tags_applied[tag_name] += tagged_events_counter
                             except elasticsearch.TransportError \
```

Comment:
Can you take the opportunity to refactor so we don't need the "\" . Just to stay consistent with the rest of the codebase.

Reply:
Not sure if I understand which " you mean? You mean: ```query_string='tag:"sigma*"')``` ?

Reply:
damn it, GH removed my \

Reply:
I meant, remove the / :)